### ISSUE TITLE
Strange create VAG consideration #4

### ISSUE THREAD

**Issue by scaleoutsean** [NetAppDocs/element-software#29]
![image](https://user-images.githubusercontent.com/16622642/125888188-1befe97f-cb2c-49be-bc3b-ca7129b17cab.png)

Page: [CreateVolumeAccessGroup](https://docs.netapp.com/us-en/element-software/api/reference_element_api_createvolumeaccessgroup.html)

I don't think SolidFire lets you create a "VAG of VAGs".

How it actually works is "each volume can belong to a maximum of four ~~other~~ volume access groups."  

Edit 1: I crossed "other" because it's not 1 + "four other". It's just four.

![image](https://user-images.githubusercontent.com/16622642/125888363-cffd35d8-1120-4b4a-a771-0d19ab4614f6.png)

This problem appears in various other docs, e.g. the SDKs, so please check the SDKs and other docs as well.

http://solidfire.github.io/sdk-dotnet/help//html/T_SolidFire_Element_Api_VolumeAccessGroup.htm

Edit 2: I found a page with correct info

![image](https://user-images.githubusercontent.com/16622642/125892140-999ed812-7f75-4a83-a850-1f308985d9b9.png)

https://docs.netapp.com/us-en/element-software/storage/concept_data_manage_vol_access_group_work_with_volume_access_groups_and_initiators.html

**Comment by netapp-dbagwell** [comment]
Thanks, Sean. Yeah, that doesn't make much sense. Created a doc ticket for this. We'll correct that throughout the doc set where possible.

**Comment by netapp-pcarriga** [comment]
@scaleoutsean - Thanks for the feedback, Sean, and apologies for the delayed response.
I've looked into where updates are possible in the doc set and, at this time, it is only possible to publish the correction in the doc.netapp.com documentation:
[CreateVolumeAccessGroup in Element 12.5](https://docs.netapp.com/us-en/element-software-/api/reference_element_api_createvolumeaccessgroup.html)
[CreateVolumeAccessGroup in Element 12.3](https://docs.netapp.com/us-en/element-software-123/api/reference_element_api_createvolumeaccessgroup.html)
Thanks again.
Closing this ticket.
